simplq/src/components/pages/Home/index.jsx

```diff
@@ -1,10 +1,26 @@
 import React from 'react';
-import { Landing, BenefitsInfo, HowToCreate, HowToJoin, ExtraInfo } from './StaticInfos';
+import { BenefitsInfo, HowToCreate, HowToJoin, ExtraInfo } from './StaticInfos';
+import Header from '../../common/Header';
+import styles from '../../../styles/homePage.module.scss';
+import CreateJoinForm from './CreateJoinForm';
 
 const Home = (props) => {
   return (
     <>
-      <Landing history={props.history} />
+      <div className={styles['landing-page']}>
+        <div>
+          <Header className={styles['main-header']} text="SimplQ" />
+          <h3 className={styles.center}>A long overdue alternative to physical Queues</h3>
+        </div>
+        <CreateJoinForm history={props.history} />
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320">
+          <path
+            fill="#6C63FF29"
+            fillOpacity="1"
+            d="M0,224L80,192C160,160,320,96,480,101.3C640,107,800,181,960,181.3C1120,181,1280,107,1360,69.3L1440,32L1440,320L1360,320C1280,320,1120,320,960,320C800,320,640,320,480,320C320,320,160,320,80,320L0,320Z"
+          />
+        </svg>
+      </div>
       <BenefitsInfo />
```

Comment:
Could you move this landing page div into another component and import it here? I guess it doesn't make sense to put it in staticInfos anymore. I feel it looks ugly the way it is right now.

Reply:
It's not in staticInfos anymore. I moved it into the index.jsx file. If you feel we need to create a separate component for the landing page (a separate page) and move it out of index.jsx, I can. What do you say?

Reply:
Actually you are right, a separate component will be a good idea. I moved it out

https://github.com/SimplQ/simplQ-frontend/commit/d7d953c83795a7d3c4e797aab8b04a69c37cf8c4